packages/hs-react-ui/src/components/Card/Card.tsx

```diff
@@ -20,6 +33,19 @@ export const CardContainer = styled(Div)`
       transition: filter ${timings.slow}, box-shadow ${timings.slow}, border ${timings.normal};
       ${getShadowStyle(elevation, colors.shadow)}
       background-color: ${colors.background};
+      ${
+        feedbackType === FeedbackTypes.simple && !disableFeedback
+          ? `
+      &:active {
+        background-color: ${
+          colors.background !== 'transparent'
+            ? darken(0.1, colors.background)
+            : 'rgba(0, 0, 0, 0.1)'
+        };
+      }
+    `
+          : ''
+      }
```

Comment:
Does this mean that when the card is clicked that it'll darken? Is that a behavior that we want? I know this came from the button's CSS but I'm thinking that kind of behavior would look a little odd on the card.

Reply:
yes, it does. The ticket was to provide interaction feedback when the card is clicked and only when an onClick function is provided by the end user (so the default onClick doesn't cause it). I saw what you did with the button's interaction feedback making it simple or ripple and did the same here to be consistent. The card defaults to ripple. There is a prop to disable the feedback entirely as well `disableFeedback` that is just a boolean. I'm happy to talk about what we expect!